### DISCUSSION TITLE
Passing dataclass JSON between tasks

### DISCUSSION THREAD

**Discussion by samuel-sujith** [flyteorg/flyte#2802] [General]
It seems that whenever I am passing a JSON dataclass between tasks that have infinity values in float type it gets automatically converted to the string "Infinity" instead of preserving type. 

Do you know if this is a known limitation of using JSON dataclasses or is there something related to flyte passing json values with infinity float values.

**Comment by Future-Outlier** [comment]
I think it will be solved by this issue.
https://github.com/flyteorg/flyte/issues/5318
